### PR TITLE
Cologne parking data

### PR DESCRIPTION
**Pull request by bergie**
Add a second data source: parking spaces in Cologne, Germany

7b5f15ce-2696-4132-84be-dec4367408e4

### CODE REVIEW

**Comment by jonnor** [comment]
Any changes needed in the graph used by the Docker image?

**Comment by bergie** [comment]
@jonnor on web server side, no changes. For fetching data, we need to trigger also the `UpdateCologneParkingData` graph periodically

**Comment by jonnor** [comment]
So we store the data in the same place/format as the DB parking data?
Do we support fetching from a different place / different method? How would that be done?

**Comment by bergie** [comment]
@jonnor data is stored in similar format, but under different Redis geolocated set. So this means each offering only serves their own data.

If we had other data sources, they'd need similar handling (their own fetch/save graph, new offering registration)

**Comment by jonnor** [comment]
Not really part of this PR, but I would like to see another data provider enabled in the same service, to be sure that works. For example the Weather provider

File: components/CategoryToNumber.js [hunk 1]
```diff
@@ -7,14 +7,26 @@ exports.getComponent = function () {
...
     datatype: 'number',
   });
   c.process((input, output) => {
-    if (!input.hasData('in')) {
+    if (!input.hasData('in', 'dataset')) {
       return;
     }
     const data = input.getData('in');
+    const dataset = input.getData('dataset');
+    if (dataset !== 'parkspaces') {
```
Comment (jonnor) [inline components/CategoryToNumber.js:23]: Hiding this logic inside the component when it needs specific info from outside to tell it how to behave seems unnecessary obscure?

**Review by jonnor — APPROVED** [review]
Merge and deploy at will

**Pull request merged**